src/test.sol

```diff
@@ -14,32 +16,38 @@
 pragma solidity >=0.4.23;
 
 contract DSTest {
-    event eventListener          (address target, bool exact);
+    event log                    (string);
     event logs                   (bytes);
+
+    event log_address            (bytes32);
     event log_bytes32            (bytes32);
-    event log_named_address      (bytes32 key, address val);
-    event log_named_bytes32      (bytes32 key, bytes32 val);
-    event log_named_decimal_int  (bytes32 key, int val, uint decimals);
-    event log_named_decimal_uint (bytes32 key, uint val, uint decimals);
-    event log_named_int          (bytes32 key, int val);
-    event log_named_uint         (bytes32 key, uint val);
-    event log_named_string       (bytes32 key, string val);
-
-    bool public IS_TEST;
+    event log_int                (int);
+    event log_uint               (uint);
+    event log_bytes              (bytes);
+    event log_string             (string);
+
+    event log_named_address      (string key, address val);
+    event log_named_bytes32      (string key, bytes32 val);
+    event log_named_decimal_int  (string key, int val, uint decimals);
+    event log_named_decimal_uint (string key, uint val, uint decimals);
+    event log_named_int          (string key, int val);
+    event log_named_uint         (string key, uint val);
+    event log_named_bytes        (string key, bytes val);
+    event log_named_string       (string key, string val);
+
+    bool public IS_TEST = true;
     bool public failed;
 
-    constructor() internal {
-        IS_TEST = true;
-    }
+    address constant HEVM_ADDRESS =
+        address(bytes20(uint160(uint256(keccak256('hevm cheat code')))));
+
+    modifier mayRevert() { _; }
+    modifier testopts(string memory) { _; }
```

Comment:
Should we leave these out until we actually support them in `hevm`?

Reply:
imo no because upgrading ds-test is a pain